docs/refmanual.rst

```diff
@@ -6,4 +6,5 @@ Cascade-AT Reference Manual
    :maxdepth: 2
 
    core
+   model
```

Comment:
Should these be listed in alphabetical order, or does the order start at core and go out from there, somehow?

Reply:
Hmm. I didn't even think about order. I was just cramming things in willy nilly. I think you could make a case for either most general -> most specific or alphabetical as the correct order. Do we have an established pattern?